tests/lib/agent.js

```diff
@@ -29,16 +29,20 @@ class Agent {
   // Add authentication
   auth () {
     this.userName = process.env.TERMINUSDB_USER
-    assert(this.userName, 'Missing environment variable: TERMINUSDB_USER')
```

Comment:
Why do you remove this?

Reply:
Because the JWT auth does not need a user, since the user is already encoded in the token.

Reply:
But that's only for JWT. `this.userName` is used twice below. What happens when it's `undefined` in those cases?

```js
      this.agent.use((request) => {
        request.set(insecureUserHeader, this.userName)
      })
```

```js
      this.agent.use((request) => {
        request.auth(this.userName, pass)
      })
```

Reply:
You're right. It should conditionally assert it instead. :smile:

Reply:
I think I fixed it in the latest commit.